org-roam.el

```diff
@@ -644,40 +644,73 @@ Throw an error if multiple choices exist."
-             collect (pcase-let* ((`(,file ,level ,pos ,alias ,title ,id ,properties ,olp) row)
-                                  (`(,file-hash ,file-atime ,file-mtime) (gethash file files-table))
-                                  (node (org-roam-node-create :id id
-                                                              :level level
-                                                              :file file
-                                                              :file-hash file-hash
-                                                              :file-atime file-atime
-                                                              :file-mtime file-mtime
-                                                              :title alias
-                                                              :point pos
-                                                              :properties properties
-                                                              :olp olp
-                                                              :tags (gethash id tags-table)))
-                                  (candidate-main (org-roam-node--format-entry node (1- (frame-width))))
-                                  (tag-str (org-roam--tags-to-str (org-roam-node-tags node))))
-                       (cons (propertize (concat candidate-main
-                                                 (propertize tag-str 'invisible t))
-                                         'node node)
-                             node)))))
+  (let ((rows (org-roam-db-query
+   "SELECT id, file, title, \"level\", todo, pos, priority, scheduled, deadline,
+           properties, olp, atime, mtime,
+           '(' || group_concat(tags,  ' ') || ')' as tags, aliases, refs
+    FROM
+     (SELECT
+       nodes.id as id,
+       nodes.file as file,
+       nodes.\"level\" as \"level\",
+       nodes.todo as todo,
+       nodes.pos as pos,
+       nodes.priority as priority,
+       nodes.scheduled as scheduled,
+       nodes.deadline as deadline,
+       nodes.title as title,
+       nodes.properties as properties,
+       nodes.olp as olp,
+       files.atime as atime,
+       files.mtime as mtime,
+       tags.tag as tags,
+       '(' || group_concat(aliases.alias, ' ') || ')' as aliases,
+       '(' || group_concat(refs.ref, ' ') || ')' as refs
+     FROM nodes
+     LEFT JOIN files ON files.file = nodes.file
+     LEFT JOIN tags ON tags.node_id = nodes.id
+     LEFT JOIN aliases ON aliases.node_id = nodes.id
+     LEFT JOIN refs ON refs.node_id = nodes.id
+     GROUP BY nodes.id, tags.tag )
+   GROUP BY id, aliases, refs;")))
+    (cl-loop for row in rows
+             append (pcase-let* ((`(,id ,file ,title ,level ,todo ,pos ,priority ,scheduled
+                                        ,deadline ,properties ,olp ,atime ,mtime ,tags ,aliases, refs) row)
+                                 (all-titles (cons title aliases))
+                                 (nodes (mapcar (lambda (t)
```

Comment:
This will error with `Attempt to set a constant symbol: t`, because `t` and `nil` are self-evaluating symbols in Elisp.